dt/Compute/DFG-compute-18.0-3cont-2comp-3cpeh-ipv4-geneve-ceph-nfs-fips

```diff
@@ -0,0 +1,56 @@
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## General information
+
+| Revision | Change              |  Date   |
+| -------: | :------------------ | :-----: |
+|     v0.1 | Initial publication | 2023-12-14 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+| Ceph                                       | vm    | 3      |
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+| --------- | ---------------------------- | ------------------ |
+| Neutron   | ML2/OVN, Geneve              | Must have          |
+| Compute   | set-nova-scheduler-filter    | Must have          |
+| Glance    | Ceph/RBD                     | Must have          |
+| Cinder    | NFS                          | Must have          |
+| Keystone  |  Any                         | Must have          |
```

Comment:
also, which services are really important and which are "supporting" the main goal of the topology?
For example, keystone is always needed, but not for testing

Reply:
tried following the new template and updated.

Reply:
Imo, it's useful to highlight which services a *required* (as opposed to all the services) if we want to merge multiple DTs into a single job in the future as part of some Consolidation effort.

Reply:
it may be important that they are not merged as that might change the behaviour in some cases.

Reply:
That makes it eve more important to know which services can affect the outcome of the deployment and testing, not just avoid combining them because something may change.